projects/view/client/activeProjectDetail/activeProjectDetail.js

```diff
@@ -0,0 +1,64 @@
+// refactored too singleProjectDetails.js and rtProjectInfoSection.js 
+Template.activeProjectDetail.onRendered(function () {
+	$('.summary').after('<span id="show-more"> More...</span>');
+	//refactored too singleProjectDetailed.js using meteors event handlers
+	$('#show-more').click(function(){
```

Comment:
Move these jQuery events into the template event map. For a reference, see the `Template.singleProjectDetailed.events({})`